### ISSUE TITLE
Problem with Jira integration "No module named 'pbr'"

### ISSUE THREAD

**Issue by trenerok** [dunbarcyber/cyphon#472]
Hi! i have problem with Jira integration. When i push to create jira issue from alert i have some problem:

```
**docker logs -f cyphondock_cyphon_1
Running Production Server
ERROR Internal Server Error: /api/v1/actions/1/run/
Traceback (most recent call last):
File "/usr/local/lib/python3.6/site-packages/django/core/handlers/exception.py", line 41, in inner
response = get_response(request)
File "/usr/local/lib/python3.6/site-packages/django/core/handlers/base.py", line 187, in _get_response
response = self.process_exception_by_middleware(e, request)
File "/usr/local/lib/python3.6/site-packages/django/core/handlers/base.py", line 185, in _get_response
response = wrapped_callback(request, callback_args, **callback_kwargs)
File "/usr/local/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
return view_func(args, kwargs)
File "/usr/local/lib/python3.6/site-packages/rest_framework/viewsets.py", line 95, in view
return self.dispatch(request, *args, kwargs)
File "/usr/local/lib/python3.6/site-packages/rest_framework/views.py", line 494, in dispatch
response = self.handle_exception(exc)
File "/usr/local/lib/python3.6/site-packages/rest_framework/views.py", line 454, in handle_exception
self.raise_uncaught_exception(exc)
File "/usr/local/lib/python3.6/site-packages/rest_framework/views.py", line 491, in dispatch
response = handler(request, args, *kwargs)
File "/usr/src/app/cyphon/responder/actions/views.py", line 72, in run
dispatch = action.get_dispatch(user=request.user, alert=alert)
File "/usr/src/app/cyphon/responder/actions/models.py", line 121, in get_dispatch
transport = self.create_request_handler(user=user)
File "/usr/src/app/cyphon/ambassador/endpoints/models.py", line 176, in create_request_handler
module = self._get_module()
File "/usr/src/app/cyphon/ambassador/endpoints/models.py", line 161, in _get_module
module = importlib.import_module(module_full_name)
File "/usr/local/lib/python3.6/importlib/init.py", line 126, in import_module
return _bootstrap._gcd_import(name[level:], package, level)
File "<frozen importlib._bootstrap>", line 994, in _gcd_import
File "<frozen importlib._bootstrap>", line 971, in _find_and_load
File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
File "<frozen importlib._bootstrap_external>", line 678, in exec_module
File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
File "/usr/src/app/cyphon/platforms/jira/handlers.py", line 26, in <module>
import jira
File "/usr/local/lib/python3.6/site-packages/jira/init.py", line 4, in <module>
from pbr.version import VersionInfo
ModuleNotFoundError: No module named 'pbr**
```

**Comment by trenerok** [comment]
I think problem with Jira package - https://github.com/pycontribs/jira/pull/504

RESOLVED:
for fix - upgrade your jira package. In container "cyphondock_cyphon_1" - > " pip install --upgrade jira" .
**I think you should be update jira package in cyphon sources.**

**Comment by chasepbrewer** [comment]
This has been resolved in the latest release: 1.6.2. The docker image should be updated with it. Have you install through cyphondock, the VM image, or through a manual install?

**Comment by lhadjchikh** [comment]
Fixed in [1.6.2](https://github.com/dunbarcyber/cyphon/tree/1.6.2]).
